dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/SpringBindingTest.java

```diff
@@ -52,10 +52,7 @@ class SpringBindingTest {
 
   @BeforeEach
   public void setUp() {
-    mockMvc =
-        MockMvcBuilders.standaloneSetup(new BindingController())
-            .setControllerAdvice(new CrudControllerAdvice())
-            .build();
+    mockMvc = MockMvcBuilders.standaloneSetup(new BindingController()).build();
```

Comment:
There is enum related code in the `CrudControllerAdvice`. Not sure what it does but I wonder if `CrudControllerAdvice` should stay in here or the code in `CrudControllerAdvice` relating enums should be removed if it has no influence. Either way maybe do this in a separate PR.